spec/controllers/physical_rack_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe PhysicalRackController do
+  render_views
+
+  let!(:server) { EvmSpecHelper.local_miq_server(:zone => zone) }
+  let(:zone) { FactoryGirl.build(:zone) }
+
+  before(:each) do
+    stub_user(:features => :all)
+    EvmSpecHelper.create_guid_miq_server_zone
+    login_as FactoryGirl.create(:user)
+    ems = FactoryGirl.create(:ems_physical_infra)
+    @physical_rack = FactoryGirl.create(:physical_rack,
```

Comment:
This is an antipattern, please use [let](http://www.betterspecs.org/#let) instead.